resources/langfiles/lang/en_us.lang

```diff
@@ -2156,6 +2156,7 @@ susy.material.platinum_free_pgm_solution=Platinum-Free PGM Solution
 susy.material.chlorate_treated_pgm_solution=Chlorate-Treated PGM Solution
 susy.material.palladium_extract=Palladium Extract
 susy.material.diisobutylene_mixture=Diisobutylene Mixture
+susy.material.rh_ir_extraction_mixture=Rhodium-Iridium Extraction Mixture
```

Comment:
why is this a thing? Currently, tributyl phosphate is being used to separate Rh and Ir ingame.

Reply:
oh i just saw an unlocalized thing lmao